packages/client/src/core/util/utils.test.ts

```diff
@@ -1,39 +1,40 @@
+import { describe, expect, it, vi } from 'vitest';
 import { LIB_VERSION } from '../../version';
 import { checkErrorForInvalidRequestArgs, fetchRPCRequest } from './utils';
 import { standardErrors } from ':core/error';
 
 // @ts-expect-error-next-line
 const invalidArgsError = (args) =>
-  standardErrors.rpc.invalidRequest({
+  standardErrors.rpc.invalidParams({
     message: 'Expected a single, non-array, object argument.',
     data: args,
   });
 // @ts-expect-error-next-line
 const invalidMethodError = (args) =>
-  standardErrors.rpc.invalidRequest({
+  standardErrors.rpc.invalidParams({
     message: "'args.method' must be a non-empty string.",
     data: args,
   });
 // @ts-expect-error-next-line
 const invalidParamsError = (args) =>
-  standardErrors.rpc.invalidRequest({
+  standardErrors.rpc.invalidParams({
     message: "'args.params' must be an object or array if provided.",
     data: args,
   });
```

Comment:
There was a mismatch between the returned and expected error code here which was not picked up in jest tests for some reason